searchlib/src/vespa/searchlib/features/debug_wait.cpp

```diff
@@ -22,29 +19,19 @@ class DebugWaitExecutor : public FeatureExecutor
     void execute(uint32_t docId) override;
 };
 
-DebugWaitExecutor::DebugWaitExecutor(const IQueryEnvironment &env, const DebugWaitParams &params)
+DebugWaitExecutor::DebugWaitExecutor(const IQueryEnvironment &, const DebugWaitParams &params)
     : _params(params)
 {
-    (void)env;
 }
 
+using namespace std::chrono;
+
 void
 DebugWaitExecutor::execute(uint32_t)
 {
-    FastOS_Time time;
-    time.SetNow();
-    double millis = _params.waitTime * 1000.0;
-
-    while (time.MilliSecsToNow() < millis) {
-        if (_params.busyWait) {
-            for (int i = 0; i < 1000; i++)
-                ;
-        } else {
-            int rem = (int)(millis - time.MilliSecsToNow());
-            std::this_thread::sleep_for(std::chrono::milliseconds(rem));
-        }
-    }
-    outputs().set_number(0, 1.0e-6 * time.MicroSecsToNow());
+    steady_clock::time_point start = steady_clock::now();
```

Comment:
this time interval measurement could use StopWatch

Reply:
Fixed